src/openforms/contrib/haal_centraal/tests/factories.py

```diff
@@ -0,0 +1,56 @@
+import os
+from pathlib import Path
+
+import factory
+from factory.django import DjangoModelFactory, FileField
+from simple_certmanager.constants import CertificateTypes
+from zgw_consumers.constants import AuthTypes
+
+from zgw_consumers_ext.tests.factories import ServiceFactory
+
+from ..constants import BRPVersions
+from ..models import HaalCentraalConfig
+
+X_API_KEY = os.getenv("HC_BRP_PERSONEN_API_KEY", "placeholder_key")
```

Comment:
```suggestion
HC_BRP_PERSONEN_API_KEY = os.getenv("HC_BRP_PERSONEN_API_KEY", "placeholder_key")
```
with the factories being "shared" I think namespacing needs to happen at this level